tests/Unit/NumericalAlgorithms/Spectral/Test_Projection.cpp

```diff
@@ -101,3 +101,160 @@ SPECTRE_TEST_CASE("Unit.Numerical.Spectral.Projection.p.element_to_mortar",
     }
   }
 }
+
+namespace {
+DataVector to_upper_half(const DataVector& p) noexcept {
+  return 0.5 * (p + 1.);
+};
+
+DataVector to_lower_half(const DataVector& p) noexcept {
+  return 0.5 * (p - 1.);
+};
+
+// `to_element_self` (`to_element_other`) is the function mapping the
+// [-1,1] interval to the half that we are (are not) interpolating
+// from.
+template <typename F1, typename F2>
+void check_mortar_to_element_projection(const Matrix& projection,
+                                        F1&& to_element_self,
+                                        F2&& to_element_other) noexcept {
+  // Notation:
```

Comment:
disambiguate / clarify that the is for the variable names defined below (as opposed to the argument names above)